includes/storage-quickstart-tutorial-intro-include-powershell.md

```diff
@@ -17,28 +17,30 @@ Connect-AzAccount
 If you don't know which location you want to use, you can list the available locations. Display the list of locations by using the following code example and find the one you want to use. This example uses **eastus**. Store the location in a variable and use the variable so you can change it in one place.
 
 ```powershell
```

Comment:
When a code sample uses Azure PowerShell commands, we use the `azurepowershell` code block label. If it's a command that can be run from within Azure Cloud Shell, then we use the `azurepowershell-interactive` code block label. Most Azure PowerShell commands can be run from within Cloud Shell.

Reply:
So I have updated this file to use azurepowershell-interactive and committed it.